packages/web-components/src/tree-item/tree-item.base.ts

```diff
@@ -114,6 +114,12 @@ export class BaseTreeItem extends FASTElement {
       return;
     }
 
+    //If a tree item is nested and initially set to selected expand the tree items so the selected item is visible
+    const selectedItems = this.querySelectorAll('[selected]');
+    if (selectedItems.length > 0) {
+      this.expanded = true;
+    }
```

Comment:
Should this be guarded to only work on tree items?

```suggestion
    this.expanded = Array.from(this.querySelectorAll('*')).some(el => isTreeItem(el) && el.selected);
```

Reply:
I don't think its neccessary, but I do like the one line approach... I'll update.